metricbeat/module/kubernetes/state_cronjob/state_cronjob_integration_test.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func TestFetchMetricset(t *testing.T) {
-	config := test.GetKubeStateMetricsConfig(t, "state_cronjob")
+	config := test.GetKubeStateMetricsConfigWithMetaDisabled(t, "state_cronjob")
```

Comment:
why for cronjob only `add_metadata` must be disabled? as I see for other k8s resources used default value - `true`

Reply:
Cause in version before v1.21 the the current stable api is not available and hence we cannot support metadata in tests for v1.20